versions/3.0.4.md

```diff
@@ -1127,6 +1127,10 @@ In order to support common ways of serializing simple parameters, a set of `styl
 
 See [Appendix E](#appendix-e-percent-encoding-and-form-media-types) for a discussion of percent-encoding, including when delimiters need to be percent-encoded and options for handling collisions with percent-encoded data.
 
+Note that parameter data can contain characters used as delimiters by certain `style` and `explode` configurations.
+For example, when using `style: "simple"`, the arrays `["monospace", "serif", "Arial,Helvetica,sans-serif"]` and `["monospace", "serif", "Arial", "Helvetica", "sans-serif"]` would both be encoded as `"monospace,serif,Arial,Helvetica,sans-serif"`.
```

Comment:
Style "simple" explicitly references [RFC6570, section 3.2.2](https://datatracker.ietf.org/doc/html/rfc6570#section-3.2.2) which has an example for encoding commas in parameter values as `%2C`:

       {keys}             semi,%3B,dot,.,comma,%2C
       {keys*}            semi=%3B,dot=.,comma=%2C

This example and the explicitly referenced [section 3.2.1](https://datatracker.ietf.org/doc/html/rfc6570#section-3.2.1) imply that this example is encoded as `monospace,serif,Arial%2CHelvetica%2Csans-serif`.

There is no need and actually no lee-way for API providers to document something different.

I think this paragraph and the next should be removed from this PR.

We could (and probably should) extend the [style examples](https://github.com/OAI/OpenAPI-Specification/blob/v3.1.1-dev/versions/3.1.1.md#style-examples) and change one of the color values to contain a comma, for example `"black,shiny"`.

Reply:
@ralfhandl oh good catch!  I'm going to take this part ouot of this PR and go read up and see what makes sense to do.  I'll submit another PR assuming we should still say something.